netx/httptransport/httptransport.go

```diff
@@ -7,6 +7,13 @@ import (
 	"net/http"
 )
 
+// Config contains the configuration assumed by this package
```

Comment:
```suggestion
// Config contains the configuration required for constructing an HTTP transport
```

This seems more explicit?